pkgs/mime/tool/mime_types.dart

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:mime/mime.dart';
+import 'package:mime/src/default_extension_map.dart';
```

Comment:
I'd add a `show defaultExtensionMap` here.
I prefer to do that when reaching into `src/` to make it clear which private parts are being accesed. (Also because someone adding a new name to a `src/` library won't expect that it can conflict with anything.)